resources/us_autocompletions/responses/us_autocompletions.yml

```diff
@@ -0,0 +1,63 @@
+description: Returns a US autocompletion object.
+
+headers:
+  ratelimit-limit:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-limit"
+  ratelimit-remaining:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-remaining"
+  ratelimit-reset:
+    $ref: "../../../shared/headers/ratelimit.yml#/ratelimit-reset"
+
+content:
+  application/json:
+    schema:
+      type: object
+      properties:
+        id:
+          $ref: "../../../shared/attributes/model_ids/us_auto_id.yml"
+
+        suggestions:
+          type: array
+          description: >
+            An array of objects representing suggested addresses.
+          minItems: 0
+          maxItems: 10
+          items:
+            $ref: "../models/suggestions.yml"
+          readOnly: true
```

Comment:
we're not using `readOnly` as it is going away in `3.1.0`

Reply:
Ah yup, that's copypasta